test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -62,7 +62,33 @@ public string[] GetInstallCommandsFromLocal(string path)
 
         public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
         {
-            throw new NotImplementedException("aziot-edge and aziot-identity-service currently aren't available in package repos");
+            string debPackages = $"https://packages.microsoft.com/config/{this.os}/{this.version}/multiarch/prod.list";
+            if (this.os.ToLower() == "ubuntu" && this.version == "20.04")
```

Comment:
As mentioned in other thread, I don't think checking against 20.04 is necessarily the right thing as if I used 20.10 this would not work... Could we just default if version is not present?

Reply:
ah, good point for 20.10... I'll see what I can do...